1_fetch/src/get_sb_data.R

```diff
@@ -85,24 +85,63 @@ make_dl_table <- function(idtostart, outdir) {
   rownames(tabledl) <- NULL
   
   #if you would like to export the table as csv
-  filepath <- file.path(outdir, paste0("sb_dl_table.csv"))
+  filepath <- file.path(outdir, paste0("sb_table_full.csv"))
   write.csv(tabledl,filepath, row.names = FALSE)
```

Comment:
Can you check the code for write.csv and read.csv, and change to use write_csv and read_csv?
```suggestion
  write.csv(tabledl,filepath, row.names = FALSE)
```

Reply:
Addressed